webserver/include/webserver/pluginhelper.h

```diff
@@ -59,7 +62,7 @@ inline void registerPluginSimple(
         << "Skip project '" << project << "', because no database "
         << "connection string can be found for it in the '"
         << projectInfo << "' file!";
-      throw std::runtime_error("No db connection string!");
+      continue;
```

Comment:
There is also one place below in this for loop where we can throw an exception:
https://github.com/Ericsson/CodeCompass/blob/e5e3acda7a8be21947fb583ac99a9f3ae73823d9/webserver/include/webserver/pluginhelper.h#L74